Contributing.md

```diff
@@ -1,24 +1,24 @@
-# How to Contribute
+# How to Contribute in Buzzphonics
```

Comment:
Thank you for making these changes. For the brand name, it starts with a lowercase b. Can you this to "to buzzphonics" please?